### PR TITLE
Add 'homey tools' to the commands to have a shortcut to the developer tools

### PR DESCRIPTION
**Pull request by Christian-Athom**

### CODE REVIEW

File: bin/cmds/tools.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const opn = require('opn');
+const { Log } = require('../..');
+const config = require('../../config.js');
+
+exports.desc = 'Homey Developer Tools';
```
Comment (RobinBol) [inline bin/cmds/tools.js:7]:
```suggestion
exports.desc = 'Open Homey Developer Tools';
```

**Review by RobinBol — APPROVED** [review]
Nice! (removed the other commits, seems you created this branch based on production)

**Review by tjallingt — APPROVED** [review]
I like it, nice shortcut that binds our tools together. Maybe we should add `homey docs` too?

**Comment by RobinBol** [comment]
> I like it, nice shortcut that binds our tools together. Maybe we should add `homey docs` too?

Would be nice, would that open apps.developer.homey.app or the SDK reference? If the former, maybe `homey docs sdk` (or something similar) could also be added.

**Comment by tjallingt** [comment]
I think I'd expect `homey docs` to open https://apps.developer.homey.app/

Not sure what command I would expect to open the API reference, maybe `homey api`, `homey docs api`, `homey sdk` or `homey docs sdk`?

**Comment by RobinBol** [comment]
> I think I'd expect `homey docs` to open https://apps.developer.homey.app/
> 
> Not sure what command I would expect to open the API reference, maybe `homey api`, `homey docs api`, `homey sdk` or `homey docs sdk`?
> 
> Maybe if we add "too many" links it makes sense to namespace these under `homey open <target>` or something?

Hm yes, not sure if adding the sdk command is very valuable, you already have the typescript definitions and using `homey docs` you are 1 click away from the sdk reference anyway.

**Comment by Christian-Athom** [comment]
Should i add the docs command to?

**Review by tjallingt — APPROVED** [review]

**Pull request merged**